### PR TITLE
Object oriented with the 'class' keyword

### PR DESCRIPTION
**Pull request by Oyekunle-Mark**

### CODE REVIEW

File: assignments/prototype-refactor.js [hunk 1]
```diff
@@ -1,9 +1,165 @@
...
+  }
+
+  takeDamage(points) {
+    this.healthPoints -= points;
+    return `${this.name} took damage.\n`;
+  }
+}
+
+class Humanoid extends CharacterStats {
+  constructor(humanObj) {
```
Comment (GianBarGian) [inline assignments/prototype-refactor.js:34]: Well done at refactoring your code with classes. Your code is really well written.
I like how meaningful was your parameter name here, a small thing like this make your code even more readable.

File: assignments/lambda-classes.js [hunk 2]
```diff
@@ -1 +1,191 @@
...
+    );
+    this.previousBackground = studentAttr.previousBackground;
+    this.className = studentAttr.className;
+    this.favSubjects = studentAttr.favSubjects;
+    this.grade = studentAttr.grade;
+    this.hasGraduated = studentAttr.hasGraduated;
+  }
+
+  listsSubjects() {
+    this.favSubjects.map(subject => console.log(subject));
```
Comment (GianBarGian) [inline assignments/lambda-classes.js]: You get undefined when you log the function to the console.
You could want to refactor this function to return a string with all the values of your array.

Comment (Oyekunle-Mark) [reply assignments/lambda-classes.js]: Thanks.
I have refactored it to return a string instead.

**Comment by GianBarGian** [comment]
Good job with the project and to fix up your code.

**Pull request merged**